sky/data/mounting_utils.py

```diff
@@ -19,7 +19,7 @@ def get_s3_mount_install_cmd() -> str:
     install_cmd = ('sudo wget -nc https://github.com/romilbhardwaj/goofys/'
                    'releases/download/0.24.0-romilb-upstream/goofys '
                    '-O /usr/local/bin/goofys && '
-                   'sudo chmod +x /usr/local/bin/goofys')
+                   'sudo chmod 755 /usr/local/bin/goofys')
```

Comment:
Is it because the user we create on AWS is not in the same group as the root account, causing the `sudo chmod +x` only changes the permission for the root user, but not the current user (`ubuntu`)? 
I think making it `755` should be fine, but just wondering if `chmod +x` without sudo will make it work?

Also it might be possible to add some config in the following to add the user into the same group, e.g.,
https://github.com/skypilot-org/skypilot/blob/a081bb9b0316af40e4bb9086af4276b923b0e918/sky/templates/aws-ray.yml.j2#L103-L109

Reply:
hmm `sudo chmod +x` should grant execute permissions to all three -(u)ser, (g)roup and (o)thers. I tried with just `chmod +x`, but that won't work since the owner of the file is root. We could chown then chmod, but generally files in /usr are supposed to be owned by root...